### PR TITLE
Slack integration in flip card

### PR DESCRIPTION
**Pull request by Pallavi-PH**
Signed-off-by: Pallavi-PH <pallaviph02@gmail.com>

This PR integrates "Join k8s slack" functionality

Screenshot 2021-06-22 at 3 31 12 PM

### CODE REVIEW

**Comment by IsAmrish** [comment]
this slack card is displaying on flip, right??

**Comment by Pallavi-PH** [comment]
> this slack card is displaying on flip, right??

Yes, this displays on flip

**Comment by IsAmrish** [comment]
please check for small screen also!!

**Comment by Pallavi-PH** [comment]
> please check for small screen also!!

It is made responsive
Tablet view:
Screenshot 2021-06-22 at 3 56 40 PM

Mobile view:
Screenshot 2021-06-22 at 3 56 54 PM

**Review by IsAmrish — APPROVED** [review]
lgtm

File: website/src/components/JoinCommunity/index.tsx [hunk 1]
```diff
@@ -42,44 +29,19 @@ const JoinCommunity: React.FC = () => {
...
-                                        InputLabelProps={{
-                                            className: classes.label,
-                                        }}
-                                        />
-                                        <IconButton aria-label="submit" className={classes.iconButton} disabled={disableContinueButton} type="submit">
-                                            <img loading="lazy" src="../images/svg/arrow_orange.svg" alt={t('joinCommunity.submitAlt')}/>
-                                        </IconButton>
-                                    </div>
-                                </form>
+                                <iframe src="https://slack.k8s.io/" title="abc" height="480" frameBorder="0" scrolling="no"></iframe>
```
Comment (AVRahul) [inline website/src/components/JoinCommunity/index.tsx:39]: can we update the title with some relevant word?

Comment (Pallavi-PH) [reply website/src/components/JoinCommunity/index.tsx]: Updated it. Thanks

**Review by rakeshPRaghu — APPROVED** [review]
Lgtm

**Review by AVRahul — APPROVED** [review]

**Pull request merged**